Packages/vcs/Lib/Canvas.py

```diff
@@ -74,7 +74,7 @@
 import vcs.manageElements
 
 class SIGNAL(object):
-    
```

Comment:
i think these are due to our pre-commit that refuses to have any white space at the end of lines.... VERY annoying, so I believe I ran some sed on the files to get rid of all of them massively